example.groovy

```diff
@@ -0,0 +1,5 @@
+class Example {
+    def sayHello() {
+        println 'Hello, World!'
+    }
+}
```

Comment:
📝 **[npm-groovy-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class does not end with a blank line before the closing brace  ClassEndsWithBlankLine